compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -717,7 +728,8 @@ setReductionVectorDistributionConfig(IREE::GPU::TargetAttr target,
 
   unsigned vectorSize = largestLoadSizeInBits / *bitWidth;
   unsigned threadLoads = vectorSize;
-  while ((reductionSize / vectorSize) % subgroupSize != 0) {
+  while (numDynamicReductionDims == 0 &&
```

Comment:
nit: put an `if` around this loop isntead?

Reply:
Makes sense.